spec/helpers/timelines_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the TimelinesHelper. For example:
+#
+# describe TimelinesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe TimelinesHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
It would be good to implement these to since they aren't getting covered elsewhere.